### ISSUE TITLE
Improve table accessibility

### ISSUE THREAD

**Issue by zephraph** [oxidecomputer/console#1711]
- Focus rings not shown for focused cell when navigating with arrows
- Column action menus not openable with keyboard

It's worth taking a deeper look at the experience we want in making the table accessible. What cells should be tabbable, what's announced per row, etc.

**Comment by david-crespo** [comment]
I can open column actions with the keyboard. We can tab to nav links too. I think this is ok for now.
